src/main/java/org/jabref/gui/externalfiles/ImportHandler.java

```diff
@@ -116,7 +116,7 @@ protected List<ImportFilesResultItemViewModel> call() {
 
                     try {
                         if (FileUtil.isPDFFile(file)) {
-                            var pdfImporterResult = contentImporter.importPDFContent(file);
+                            var pdfImporterResult = contentImporter.importPDFContent(file, bibDatabaseContext.getMetaData());
```

Comment:
Better use this directory:         Optional<Path> firstExistingFileDir = bibDatabaseContext.getFirstExistingFileDir(filePreferences);
 
This will make sure that you always get the "right" file directory (global or from metadata)